samples/06-prs/save-data-to-file/src/main/java/org/eclipse/dataspaceconnector/extensions/api/PrsApiExtension.java

```diff
@@ -0,0 +1,59 @@
+package org.eclipse.dataspaceconnector.extensions.api;
+
+import org.eclipse.dataspaceconnector.policy.model.Action;
+import org.eclipse.dataspaceconnector.policy.model.AtomicConstraint;
+import org.eclipse.dataspaceconnector.policy.model.LiteralExpression;
+import org.eclipse.dataspaceconnector.policy.model.Permission;
+import org.eclipse.dataspaceconnector.policy.model.Policy;
+import org.eclipse.dataspaceconnector.spi.metadata.MetadataStore;
+import org.eclipse.dataspaceconnector.spi.policy.PolicyRegistry;
+import org.eclipse.dataspaceconnector.spi.system.ServiceExtension;
+import org.eclipse.dataspaceconnector.spi.system.ServiceExtensionContext;
+import org.eclipse.dataspaceconnector.spi.transfer.flow.DataFlowManager;
+import org.eclipse.dataspaceconnector.spi.types.domain.metadata.DataEntry;
+
+import java.util.Set;
+
+import static org.eclipse.dataspaceconnector.policy.model.Operator.IN;
+
+public class PrsApiExtension implements ServiceExtension {
+
+    public static final String USE_EU_POLICY = "use-eu";
+
+    // TODO: is it needed?
+    @Override
+    public Set<String> requires() {
+        return Set.of("edc:webservice", PolicyRegistry.FEATURE);
+    }
+
+    @Override
+    public void initialize(ServiceExtensionContext context) {
+
+        var dataFlowMgr = context.getService(DataFlowManager.class);
+        var flowController = new PrsApiFlowController(context.getMonitor(), new PrsApiCaller(), new FileSystemDataWriter(context.getMonitor()));
+        dataFlowMgr.register(flowController);
+        context.getMonitor().info("PRS API extension initialized!");
+
+        savePolicies(context);
```

Comment:
I don't get how this policy is used at the end. Maybe you can explain me? And then we can document that.